jolt-core/src/msm/mod.rs

```diff
@@ -20,138 +23,122 @@ pub type GpuBaseType<G: ScalarMul> = G::MulBase;
 
 use itertools::Either;
 
-#[derive(Debug, Clone, Copy, Eq, PartialEq, Ord, PartialOrd)]
-pub enum MsmType {
-    Zero,
-    One,
-    Small(usize),
-    Medium(usize),
-    Large(usize),
-}
-
-impl MsmType {
```

Comment:
I thought having this around made it easier to read/maintain.

Reply:
I removed it because it gets weird with `CompactPolynomial` –– the `Medium` size straddles the `u8` and `u16` size